frontend/index.html

```diff
@@ -3,83 +3,148 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/orders101-ui.css">
     <title>DIYToolWorld</title>
 </head>
 <body>
     <!-- Account management and contact us section -->
-    <section id="account">
-        <div>
-            <ul>
-                <li><a href="#account">Account</a></li>
+    <header class="main-theme">
+        <div class="account container">
+            <ul class = account-links>
+                <li><a href="#account"><img src="./assets/images/person-fill.svg">Account</a></li>
```

Comment:
Was just looking at the alignment of icon here, as discussed. I think if you apply `display: flex` to the `a` and then for example `margin-right: 0.4rem;` to the `img` we would get a desired result?